uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/leave-request-ctrl.js

```diff
@@ -0,0 +1,43 @@
+define([
```

Comment:
we already have qualified the name of sub controllers by *-request-ctrl.js , so i dont think we need additional distinction at folder level.